soh/soh/Enhancements/randomizer/3drando/hints.cpp

```diff
@@ -820,19 +830,50 @@ void CreateGregRupeeHint() {
 
   Text temp1 = Text{
     "By the way, if you're interested, I saw the shiniest %gGreen Rupee%w somewhere in%g ",
-    "",
+    "Au fait, si ça t'intéresse, j'ai aperçu le plus éclatant des %gRubis Verts%w quelque part à %g",
     ""
   };
 
   Text temp2 = {
     "%w.^It's said to have %rmysterious powers%w...^But then, it could just be another regular rupee.&Oh well.",
-    "",
+    "%w. On dit qu'il possède des pouvoirs mystérieux... Mais bon, ça pourrait juste être un autre rubis ordinaire.",
     ""
   };
 
     gregText = temp1 + area + temp2;
 }
 
+void CreateSheikText() {
+  //Get the location of the light arrows
+  auto lightArrowLocation = FilterFromPool(allLocations, [](const uint32_t loc){return Location(loc)->GetPlaceduint32_t() == LIGHT_ARROWS;})[0];
+  Text area = GetHintRegion(Location(lightArrowLocation)->GetParentRegionKey())->GetHint().GetText();
+  Text temp1 = Text{
+    "I overheard Ganondorf say that he misplaced the %rLight Arrows%w in&%g",
+    "J'ai entendu dire que Ganondorf aurait caché les %rFlèches de Lumière%w dans %g",
+    ""
+  };
```

Comment:
Is this AI generated or made by a human? If this is AI translated, could we ask our resident French translator @PurpleHato or someone else from the community to double check all the French text in the PR?

Also, I assume the german is missing because 3DS rando logic doesn't actually accept German? Might be something to note for a potential rando refactor to either implement german or at the very least make it so we don't need empty strings for German.

Reply:
This was translated by Hato. And yes, our 3d rando doesn't currently support German, opting for Spanish instead.

Reply:
Made note of the language issue in our rando refactor idea tracker.

Reply:
> Is this AI generated or made by a human? If this is AI translated, could we ask our resident French translator @PurpleHato or someone else from the community to double check all the French text in the PR?
> 
> Also, I assume the german is missing because 3DS rando logic doesn't actually accept German? Might be something to note for a potential rando refactor to either implement german or at the very least make it so we don't need empty strings for German.

Those were made by me, and I completely forgot to ask for the German translations, they could be a follow-up PR from my side

Side note for myself: Also need to add the 3DS system for dungeon prefix for French and German on everything that name a place, since we need to say for instance "l'Arbre Mojo", "Le Temple du Feu", "La Forêt Kokiri".